android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/Router.kt

```diff
@@ -141,7 +141,7 @@ protected constructor(
   public open fun detachChild(childRouter: Router<*>) {
     val isChildRemoved = children.remove(childRouter)
     val interactor = childRouter.interactor
-    ribRefWatcher.watchDeletedObject(interactor)
+    ribRefWatcher.watchDeletedObject(interactor, "detached child router ${childRouter.javaClass.name}")
```

Comment:
I made a description. You might want something different. But it's nice to be able to provide context on what's causing this to be defined as something to be GCed.